packages/cloudflare/src/pages-plugin.ts

```diff
@@ -23,7 +23,16 @@ export function sentryPagesPlugin<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Params extends string = any,
   Data extends Record<string, unknown> = Record<string, unknown>,
->(options: CloudflareOptions): PagesPluginFunction<Env, Params, Data, CloudflareOptions> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  PluginParams = any,
```

Comment:
This is weakening the type, but it's on purpose - constraining the `PluginParams` to be both an object and function doesn't work really well 😬.